static/sass/styles.scss

```diff
@@ -122,6 +122,13 @@
   }
 }
 
+.p-address-row {
```

Comment:
You can use `u-hide--large` from https://vanillaframework.io/docs/utilities/hide

Reply:
I applied that class on the `tr` and I cannot get it to be shown in any way. Also, it is inconsistent with the style just above:

```
.p-instance-table {
  @media screen and (max-width: 1400px) {
    th,
    td {
      &:nth-child(4),
      &:nth-child(5) {
        display: none;
      }
    }
  }
}
```

Reply:
You are right, I forgot we can't use large, because it is `1036px` and we don't want to change it either because it has other side effects. So I guess we have to keep it as is.